src/pages/Ingame/index.js

```diff
@@ -1,35 +1,45 @@
 import React, { useState, useEffect } from "react";
 import { useLocation, useHistory } from "react-router-dom";
 import styled from "styled-components";
-import GameItem from "../../components/GameItem";
+import GameItem from "../../components/Ingame/GameItem";
+import GameCard from "../../components/Ingame/GameCard";
 import RuleBook from "../../components/RuleBook";
 import Modal from "../../components/Modal";
-import { emitJoinTeam, socket } from "../../utils/socket";
-import { getMiddleResult, getCurrentScore, sumScores, sumResults, makeData } from "../../utils/game";
+import {
+  socket,
+  emitJoinTeam,
+  getMiddleResult,
+  getCurrentScore,
+  sumScores,
+  sumResults,
+  makeData,
+  checkSpecialRound,
+} from "../../utils";
 
 export default function Game() {
   const history = useHistory();
   const location = useLocation();
   const teamInfo = location.pathname.split(":")[1].split("-");
   const [channelId, roomId, team] = teamInfo.slice(0, 3);
   const roomName = `${channelId}-${roomId}`;
-  let round = Number(teamInfo[3]);
+  const [round, setRound] = useState(Number(teamInfo[3]));
 
   const [mycard, setMycard] = useState("");
   const [isSubmitted, setIsSubmitted] = useState(false);
   const [isChecked, setIsChecked] = useState(false);
-  const [selectBoard, setSelect] = useState(); // 전체 라운드 선택
-  const [roundScore, setRoundScore] = useState(); // 라운드별 점수
-  const [scoreBoard, setScoreBoard] = useState(); // 전체 라운드 점수
-  const [roundDone, setRoundDone] = useState(false); // 라운드 종료 체크
-  const [currentTeamScore, setCurrentScore] = useState(0); // 현재 점수
+  const [selectBoard, setSelect] = useState();
```

Comment:
이 부분이 주석내용처럼 라운드 선택부분이라고 한다면, 명시적으로 구체화 해주는 것이 좋을것 같습니다.
```suggestion
  const [selectRound, setSelectRound] = useState();
```

Reply:
해당부분은 선택한 결과판인데 좀더 명시적으로 수정해두겠습니다